WordPress/Classes/ViewRelated/Blog/CreateNewBlog/CreateNewBlogViewController.m

```diff
@@ -0,0 +1,551 @@
+#import "CreateNewBlogViewController.h"
+#import "WordPressComApi.h"
+#import "WPNUXMainButton.h"
+#import "WPNUXSecondaryButton.h"
+#import "WPWalkthroughTextField.h"
+#import "WPAsyncBlockOperation.h"
+#import "WPComLanguages.h"
+#import "WPWalkthroughOverlayView.h"
+#import "WPNUXUtility.h"
+#import "WPStyleGuide.h"
+#import "UILabel+SuggestSize.h"
+#import "WPAccount.h"
+#import "Blog.h"
+#import "WordPressComServiceRemote.h"
+#import "AccountService.h"
+#import "BlogService.h"
+#import "ContextManager.h"
+#import "NSString+XMLExtensions.h"
+#import "Constants.h"
+
+#import "WordPress-Swift.h"
+
+@interface CreateNewBlogViewController ()<UITextFieldDelegate,UIGestureRecognizerDelegate> {
+    WPNUXSecondaryButton *_cancelButton;
+    UILabel *_titleLabel;
+    UILabel *_siteAddressWPComLabel;
+    WPWalkthroughTextField *_siteTitleField;
+    WPNUXMainButton *_createBlogButton;
+    WPWalkthroughTextField *_siteAddressField;
+    
+    NSOperationQueue *_operationQueue;
+    
+    BOOL _authenticating;
+    BOOL _keyboardVisible;
+    BOOL _userDefinedSiteAddress;
+    CGFloat _keyboardOffset;
+    
+    NSDictionary *_currentLanguage;
+}
+
+@end
+
+@implementation CreateNewBlogViewController
+
+static CGFloat const CreateBlogStandardOffset             = 15.0;
```

Comment:
@kwonye these constants should be moved above `@interface` extension and after any `#import` declarations.